op-guide/ansible-deployment.md

```diff
@@ -125,31 +125,28 @@ Make sure you have logged in to the Control Machine using the `root` user accoun
 
 ## Step 3: Download TiDB-Ansible to the Control Machine
 
-1. Log in to the Control Machine using the `tidb` user account and enter the `/home/tidb` directory. The corresponding relationship between the `tidb-ansible` branch and TiDB versions is as follows:
+1. Log in to the Control Machine using the `tidb` user account and enter the `/home/tidb` directory. The relationship between `tidb-ansible` and TiDB versions is as follows:
 
-    | tidb-ansible branch | TiDB version | Note |
-    | ------------------- | ------------ | ---- |
-    | release-2.0 | 2.0 version | The latest 2.0 stable version. You can use it in the production environment. |
-    | release-2.1 | 2.1 version | The latest 2.1 stable version. You can use it in the production environment (recommended). |
-    | master | master version | This version includes the latest features with a daily update. |
+    | TiDB version | tidb-ansible tag | Note |
+    | -------- | ---------------- | --- |
+    | 2.0 version | v2.0.10, v2.0.11 | It is the latest 2.0 stable version which can be used in the production environment. |
+    | 2.1 version | v2.1.1 ~ v2.1.6 | It is the latest 2.1 stable version which can be used in the production environment (recommended). |
+    | 3.0 version | v3.0.0-beta, v3.0.0-beta.1 | It is currently a beta version which is not recommended to use in the production environment. |
+    | The latest version | None | It includes the newest features and is updated on a daily basis, so it is not recommended to use it in the production environment. |
```

Comment:
```suggestion
    | `master` branch | None | It includes the newest features and is updated on a daily basis, so it is not recommended to use it in the production environment. |
```
In the previous text it was called 'master'. I think we should keep using that name because it has meaning in the code (as does release-2.0, release-2.1). Calling it 'latest' does not convey the same meaning.

Reply:
@liubo0127 @lilin90 How do you think? Should I change it back?

Reply:
Feedback addressed. PTAL again. @morgo @lilin90

Reply:
Let's keep using `master`.